modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/java/com/liferay/osb/faro/web/internal/application/MainApplication.java

```diff
@@ -40,17 +60,62 @@ public Set<Object> getControllers() {
 		controllers.add(_channelFaroController);
 		controllers.add(_definitionsFaroController);
 		controllers.add(_issueFaroController);
-		controllers.add(_mainController);
 		controllers.add(_notificationFaroController);
 		controllers.add(_oAuth2FaroController);
 		controllers.add(_preferencesFaroController);
 		controllers.add(_projectFaroController);
 		controllers.add(_reportFaroController);
 		controllers.add(_userFaroController);
+		controllers.add(this);
 
 		return controllers;
 	}
 
+	@GET
+	@Path("/{groupId}/entities")
```

Comment:
Hey @regiCesar21, why did you do this inline? Please explain it in the commit message. Thx